codeHF/config_tasks.sh

```diff
@@ -182,6 +195,17 @@ function MakeScriptO2 {
   [ $SAVETREES -eq 1 ] && OPT_MAKECMD+=" -t"
   [ $MAKE_GRAPH -eq 1 ] && OPT_MAKECMD+=" -g"
 
+  # Make a copy of the default workflow database file before modifying it
+  if [[ $DOO2_EVSEL -eq 1 || $DOO2_CASC -eq 1 ]]; then
+    DATABASE_O2_EDIT=".${DATABASE_O2}"
```

Comment:
Define empty `DATABASE_O2_EDIT` before `if`.
Why `".${DATABASE_O2}"` instead of `"${DATABASE_O2/.yml/_edit.yml}"`?

Reply:
Fixed